indico/modules/events/papers/controllers/display.py

```diff
@@ -159,3 +162,20 @@ class RHDeletePaperComment(RHPaperCommentBase):
     def _process(self):
         delete_comment(self.comment)
         return jsonify_data(flash=False)
+
+
+class RHDisplayReviewingArea(RHPapersBase):
+    def _checkProtection(self):
+        if not session.user:
+            raise Forbidden
+        if session.user not in self.event_new.cfp.layout_reviewers and \
```

Comment:
I don't think the CFP class has `event_new` but `event` instead. Besides that: Almost. don't use the `*_reviewers` properties and check enabled first before doing an ACL check (cheaper one first to avoid doing the other one if it won't be needed)